releasenotes-dca/notes/add-exponential-backoff-to-syncinformer-9385d768f8ca21ce.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add exponential backoff for syncing with the kubernetes informers. The max retry delay can be configured with kube_cache_sync_max_retry_delay_seconds.
```

```suggestion
    Add exponential backoff for syncing with the kubernetes informers.
```

I'd prefer we avoid promoting the new config setting for now, one could configure it if advised so but let's see if it solves most of the issue without having to tinker it